jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -123,16 +115,10 @@ elif [[ ${GAZEBO_MAJOR_VERSION} -ge 11 ]]; then
     sdformat_pkg="libsdformat9-dev"
 elif [[ ${GAZEBO_MAJOR_VERSION} -ge 9 ]]; then
     sdformat_pkg="libsdformat6-dev"
-elif [[ ${GAZEBO_MAJOR_VERSION} -ge 7 ]]; then
-    sdformat_pkg="libsdformat4-dev"
 fi
 
-# Old versions used libogre-dev
-ogre_pkg="libogre-1.9-dev"
-if [[ ${IGN_RENDERING_MAJOR_VERSION} -ge 1 ]]; then
-    # support for both ogre-1.9 and ogre-2.1 was added in ign-rendering1
-    ogre_pkg="libogre-1.9-dev libogre-2.1-dev"
-fi
+# support for both ogre-1.9 and ogre-2.1 was added in ign-rendering1
+ogre_pkg="libogre-1.9-dev libogre-2.1-dev"
```

Comment:
this broke a gazebo classic CI

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gazebo-ci-pr_any-ubuntu_auto-amd64-gpu-nvidia&build=704)](https://build.osrfoundation.org/job/gazebo-ci-pr_any-ubuntu_auto-amd64-gpu-nvidia/704/) https://build.osrfoundation.org/job/gazebo-ci-pr_any-ubuntu_auto-amd64-gpu-nvidia/704/

Reply:
specifically gazebo9 because the default ci_distro in dal is xenial

Reply:
gazebo11 CI is ok

Reply:
ah shoot we can either revert this or wait for https://github.com/ignition-tooling/release-tools/pull/462

Reply:
I don't want to rush #462, here's a fix for this: https://github.com/ignition-tooling/release-tools/pull/483